### ISSUE TITLE
Add all svg/images to repo

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#419]
We need all source svg of all image files, like folder icon, in our repo.
Also the nextcloud(beta) icon should be, at least, copied in our repo.

@AndyScherzinger you did, back then, the changes. Or are the filetypes "just" copied from server? Then we should at least have a link as reference to the specific folder.
EDIT: thinking about it. I think it is better to have all images in our repo, altghough it might duplicate things, but with this approach you always have one repo which is consistent.

@jancborchardt can you assist us here?

**Comment by jancborchardt** [comment]
Anything which is modified from something available on the server repo, I would put the source here so we have it somewhere. But for the filetype icons, I would not necessarily duplicate it, because then we have check what changed.

**Comment by AndyScherzinger** [comment]
* File types are more or less just copied, 
* action/menu icons are all taken from Google's standard icon library as pngs
* launcher icon (stable/beta) are both in the promo repo
* missing: vector files for the actual what's new screens, that would be a todo for @eppfel :)

**Comment by eppfel** [comment]
An own repository or promo, for assets would be nice, than every project could embed it as a submodule. Espina2 suggested that.

@AndyScherzinger Where should I put the sauce?

**Comment by jancborchardt** [comment]
@AndyScherzinger I would say it makes a lot of sense to move the launcher icon from the promo repo to here. And adding the sources for the »what’s new« graphics is good too. The rest seems fine as it is, no need to copy sources of filetype icons or material design icons.

@eppfel both launcher icon and »what’s new« images are specific to the Android app style though. For iOS at least small modifications need to be made, and those should then be in the iOS repo.

**Comment by AndyScherzinger** [comment]
@eppfel I'd say let's put everything in here for now: https://github.com/nextcloud/android/tree/master/drawable_resources

**Comment by tobiasKaminsky** [comment]
The purpose to have all relevant icons in this repo would be, to have a better possibility to brand/theme the app.
We then could have some subfolders fileType, appIcon, ..., modify the svgs and then copy them in the real subfolders.

We will ever have to keep server and our icons in sync, nevertheless where the icons are stored.

**Comment by tobiasKaminsky** [comment]
^ @rullzer 
Just a thought how I would do it. Maybe you have a better idea.

**Comment by AndyScherzinger** [comment]
This is tricky, a central place would make things easier for maintenance of the files while we will have to do png exports anyways. That would then mean that we add a dependency to another repo. 
A wouldn't like to add a submodule though since it adds an extra steps to the dev setup which doesn't add value to the development process.

**Comment by tobiasKaminsky** [comment]
No no, no submodule.
Just a "simple" copy of the svgs files to our repository. 
With this our repository is on its own consistent and we can do a little script, that just changes all colors on the svg and do a png export.
If the server changes its icon set, we will have to manually adapt it, either copy the svg to our repository or make pngs and copy them to the correct place (as we would do it now).

So, basically no real difference, but it makes a lot of things easier for branding / theming.

**Comment by rullzer** [comment]
Having everything in the android repo would make branding and co a lot easier indeed. A submodule is tricky because if that changes at some point (and it will) we risk breaking a lot of stuff all over the places.

**Comment by AndyScherzinger** [comment]
I'd close this for now since all vectors should now be available in the Android repo.
